### ISSUE TITLE
Crash on gradle task :react-native-sodium:externalNativeBuildRelease

### ISSUE THREAD

**Issue by Looveh** [lyubo/react-native-sodium#14]
Building on Android is crashing for me in a place I've never had any problems before and I'm stumped about why.

```
...
:react-native-sodium:externalNativeBuildRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-sodium:externalNativeBuildRelease'.
> java.io.FileNotFoundException: /path/to/app/node_modules/react-native-sodium/android/.externalNativeBuild/cmake/release/mips64/android_gradle_build.json (No such file or directory)
```

Does anybody have any idea why?

**Comment by astro019** [comment]
React-Native-Sodium was working correctly in my project before.
But it crash on gradle task: react-native-sodium:externalNativeBuildRelease when I did npm install recently.

**Comment by jcking89** [comment]
@Looveh Fixed by changed gradle version to 3.1.4
I faced such error when using gradle version 3.2.0 alpha.

**Comment by lyubo** [comment]
Fixed with latest version
